Required validation rule clarification regarding whitespaces

basi01 opened the pull request:
If a string value is not empty, but consists of whitespace characters only, the Required validation rule considers it empty and thus invalid

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/mendix/docs?pullRequest=7747) <br/>All committers have signed the CLA.

basi01 committed:
Required validation rule clarification regarding whitespaces

If a string value is not empty, but consists of whitespace characters only,
the Required validation rule considers it empty and thus invalid

MariaShaposhnikova committed:
Update validation-rules.md
